runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheClientProduceFactory.java

```diff
@@ -1384,7 +1384,7 @@ private void onClientInitialFlush(
                     doClientReplyAbortIfNecessary(traceId);
                     fan.onClientFanMemberClosed(traceId, this);
                 }
-                else
+                else if (!KafkaState.initialClosed(state))
```

Comment:
IIUC, it is not possible to receive `onClientInitialFlush` if `KafkaState.initialClosed(state)` is true, agree?